host/sgx/sgxload.c

```diff
@@ -556,6 +593,10 @@ oe_result_t oe_sgx_load_enclave_data(
 #endif /* defined(OE_TRACE_MEASURE) */
 
     /* Measure this operation */
+    if (context->create_zero_base_enclave)
+    {
+        base = (uint64_t)OE_ADDRESS_ZERO;
```

Comment:
'base' is an input. Is it OK to override `base` here? Even if it's OK for the current implementation. The code is hard to understand and might cause problem if someone changes implementation.

Reply:
Fixed this bug. Validated simulation mode functioning on both current and new PSW thru tests/sim-mode.